zebra-state/src/service/check/nullifier.rs

```diff
@@ -1,12 +1,14 @@
 //! Checks for nullifier uniqueness.
```

Comment:
When the API of this module is stable, I'd like to see some unit tests for the new functions.

We should have some existing nullifier tests, but I'm not sure where they are. And they won't be using mempool transactions.

Reply:
Updates `check::nullifier` tests in [`513b811`](https://github.com/ZcashFoundation/zebra/pull/5716/commits/513b8110231debb65881e4f2c1e029ef658dd987).